content/en/_index.html

```diff
@@ -17,26 +17,49 @@
                   height="full" link_title="Get started" link_url="/docs/get-started/" %}}
   Flux is a set of continuous and progressive delivery solutions for Kubernetes, and they are open and extensible.
 
-  Some of the biggest organisations have adopted the Flux family of projects for their GitOps needs.
-  See [who is part of our community](/adopters) and how about joining yourself?
-
   The latest version of Flux brings many new features and made Flux more flexible and versatile.
   {{% /blocks/hero %}}
 
-
-  <div class="lead-gitops">
+  <div class="lead-flux">
   {{< blocks/lead >}}
 
-  <h2 id='gitops'>GitOps at your fingertips</h2>
+  <h2 id="flux-facts">Flux Facts!</h2>
```

Comment:
```suggestion
  <h2 id="flux-facts">Flux in short</h2>
```

(and if not accepted: `Flux facts!`, as we do not use PascalCase title headings elsewhere on this page)